test/units/modules/net_tools/test_nmcli.py

```diff
@@ -152,7 +187,8 @@ def mocker_set(mocker, connection_exists=False):
     """
     mocker.patch('ansible.modules.net_tools.nmcli.HAVE_DBUS', True)
     mocker.patch('ansible.modules.net_tools.nmcli.HAVE_NM_CLIENT', True)
-    get_bin_path = mocker.patch('ansible.module_utils.basic.AnsibleModule.get_bin_path')
+    get_bin_path = mocker.patch(
+        'ansible.module_utils.basic.AnsibleModule.get_bin_path')
```

Comment:
The actual `max-line-length` is 160, so there's no need of a new lines here 

https://github.com/ansible/ansible/blob/379a7f4f5a0491964c7896834f3f326412888585/tox.ini#L31